frontend/src/components/common/ConditionalButton.js

```diff
@@ -16,19 +16,30 @@ const propTypes = {
   type: PropTypes.string,
   isDropdown: PropTypes.bool,
   overlay: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+  ghost: PropTypes.bool,
+  popConfirm: shape({
+    placement: PropTypes.string,
+    title: PropTypes.string,
+    onConfirm: PropTypes.func.isRequired,
+    okText: PropTypes.string,
+    cancelText: PropTypes.string
+  })
 };
 
 const defaultProps = {
+  handleAction: null,
   string: "",
+  ghost: false,
   type: "primary",
   isDropdown: false,
+  popConfirm: null,
   overlay: "",
 };
 
 export const ConditionalButton = (props) => (
-  <div>
-    {!props.isDropdown && (
-      <Button className="full-mobile" type={props.type} onClick={props.handleAction}>
+  <span>
+    {!props.isDropdown && props.popConfirm == null && (
```

Comment:
You'll want to avoid `==` comparisons in JS. Always use `===`

But, in this case, you'll want to do:
```javascript
import { isNull } from "lodash";

!props.isDropdown && isNull(props.popConfirm) && (
```